Can't find string terminator '"' anywhere before EOF at -e line 1

My error:

```
create-dmg \
		--volname "MyApp" \
		--icon "extras/macos/icon.icns" \
		--hide-extension "dist/MyApp.app" \
		--hdiutil-quiet \
		"MyApp.dmg" \
		"dist/MyApp.app"
Creating disk image...
Mounting disk image...
Device name:     /dev/disk18
Searching for mounted interstitial disk image using /dev/disk18s... 
Mount dir:       /Volumes/dmg.YI9n9f
String found where operator expected at -e line 1, at end of line
	(Missing semicolon on previous line?)
Can't find string terminator '"' anywhere before EOF at -e line 1.
make: *** [dmg] Error 255
```

Can you help me?

Hi, thanks for having a look, with `set -x`, I've obtained the following log:

`Mounting disk image...
Device name:     /dev/disk10
Searching for mounted interstitial disk image using /dev/disk10s... 
Mount dir:       /Volumes/dmg.KKJF9v
Making link to Applications dir...
/Volumes/dmg.KKJF9v
Unknown regexp modifier "/U" at -e line 1, at end of line
Can't find string terminator '"' anywhere before EOF at -e line 1.
Error: Process completed with exit code 255.`

Thanks

--- a/create-dmg
+++ b/create-dmg
@@ -484,10 +484,10 @@ else
 					-e "s/WINH/$WINH/g" -e "s/BACKGROUND_CLAUSE/$BACKGROUND_CLAUSE/g" \
 					-e "s/REPOSITION_HIDDEN_FILES_CLAUSE/$REPOSITION_HIDDEN_FILES_CLAUSE/g" \
 					-e "s/ICON_SIZE/$ICON_SIZE/g" -e "s/TEXT_SIZE/$TEXT_SIZE/g" \
-			| perl -pe "s/POSITION_CLAUSE/$POSITION_CLAUSE/g" \
+			| perl -pe "s:POSITION_CLAUSE:$POSITION_CLAUSE:g" \
 			| perl -pe "s/QL_CLAUSE/$QL_CLAUSE/g" \
 			| perl -pe "s/APPLICATION_CLAUSE/$APPLICATION_CLAUSE/g" \
-			| perl -pe "s/HIDING_CLAUSE/$HIDING_CLAUSE/" \
+			| perl -pe "s:HIDING_CLAUSE:$HIDING_CLAUSE:" \
 			> "$APPLESCRIPT_FILE"
 
 		# pause to workaround occasional "Can’t get disk" (-1728) issues

